.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,47 @@
+name: Integration
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        opensearch_version:
+          - 1.0.0
+          - 1.0.1
+          - 1.1.0
+          - 1.2.0
+          - 1.2.1
+          - 1.2.2
+          - 1.2.3
+          - 1.2.4
+          - 1.3.0
+    steps:
+      - name: Set up JDK 14
```

Comment:
For a future PR, you should downgrade to JDK 11, or turn this into a matrix that does a combination of versions of OpenSearch and JDK, like [so](https://github.com/slack-ruby/slack-ruby-client/blob/master/.github/workflows/test.yml#L12).